components/raftstore/src/store/peer.rs

```diff
@@ -2329,7 +2333,11 @@ where
                 if self.check_disk_usages_before_propose(ctx, disk_full_opt, &mut stores) {
                     self.propose_conf_change(ctx, &req)
                 } else {
-                    let errmsg = String::from("propose failed: disk full");
+                    // let errmsg = String::from("propose failed: disk full");
+                    let errmsg = format!(
+                        "propose failed: disk full, cmd::disk_full_opt={:?}, diskUsage={:?}",
```

Comment:
Same as above.

Reply:
This is only a log,, not the drop table failed logic